Source/Fuse.Elements/Resources/Exif/Exif.uno

```diff
@@ -0,0 +1,192 @@
+using Uno;
+using Uno.IO;
+using Uno.Compiler.ExportTargetInterop;
+
+namespace Fuse.Resources.Exif
+{
+	/** ImageOrientation is used to indicate the orientation of an image.
+		The enum describes a transformation containing a vertical flip (if the FlipVertical flag is active), and then a rotation (if one of the rotation flags is active).
+		Note that the flip should be applied before the rotation.
+	 */
+	[Flags]
+	public enum ImageOrientation
+	{
+		Identity = 0,
+		Rotate90 = 1,
+		Rotate180 = 2,
+		Rotate270 = 3,
+		FlipVertical = 1 << 2,
+	}
+
+	public struct ExifData
+	{
+		public readonly ImageOrientation Orientation;
+
+		public static ExifData FromByteArray(byte[] buffer)
+		{
+			if defined(Android)
+				return ExifAndroidImpl.FromByteArray(buffer);
+			if defined(iOS)
+				return ExifIOSImpl.FromByteArray(buffer);
+			if defined(DotNet)
+				return ExifDotNetImpl.FromByteArray(buffer);
+			else
+				throw new Exception("Exif is not supported by this platform");
```

Comment:
From reviewing the call-site that uses this, I'm starting to think that this shouldn't throw, but rather return `bool` and have an out-argument, so we don't need to deal with exceptions just to be able to find out there's no EXIF information...